proposals/distribution.md

```diff
@@ -0,0 +1,65 @@
+# Abstract
+
+The Docker registry protocol has become the defacto standard across the container registry world ([https://github.com/docker/distribution/blob/master/docs/spec/api.md](https://github.com/docker/distribution/blob/master/docs/spec/api.md)).
```

Comment:
I [still][1] think we want to pin this to a specific version.  For example:

    https://github.com/docker/distribution/blob/5cb406d511b7b9163bff9b6439072e4892e5ae3b/docs/spec/api.md

[1]: https://github.com/opencontainers/tob/pull/35#discussion_r163291623

Reply:
That should be an implementation detail of the proposal. The implementors of the proposal should choose the version to adopt so this can be changed without TOB involvement.

Reply:
I think the problem is that the hyperlink will be broken when the spec is moved to the OCI distribution-spec repo.

I think keeping the hyperlink permanently after the migration is good for recording the history.

So +1 for pinning.

Reply:
And as long as the TOB-selected version isn't far behind (how far will the spec move during a week of voting?), it should be easy for the new maintainets to catch up on any subsequent drift.

Reply:
we can use such as language when pointing to a particular version...